internal/inboud_orders/repository/InboundOrdersRepository_test.go

```diff
@@ -0,0 +1,77 @@
+package repository_test
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/stretchr/testify/assert"
+	"github.com/vinigracindo/mercado-fresco-stranger-strings/internal/inboud_orders/domain"
+	"github.com/vinigracindo/mercado-fresco-stranger-strings/internal/inboud_orders/repository"
+)
+
+func TestInboundOrdersRepository_Create(t *testing.T) {
+	ctx := context.Background()
+
+	now := time.Now()
+
+	expectedInboundOrders := domain.InboundOrders{
+		Id:             1,
+		OrderDate:      now,
+		OrderNumber:    "order#1",
+		EmployeeId:     1,
+		ProductBatchId: 1,
+		WarehouseId:    1,
+	}
+
+	t.Run("should create inbound order", func(t *testing.T) {
```

Comment:
Sugestão: iniciar a frase com o nome do que está sendo testado, exemplo: "create_ok: should create inbound order"

Reply:
Done.